src/client/dfuse/ops/lookup.c

```diff
@@ -207,13 +207,13 @@ check_for_uns_ep(struct dfuse_projection_info *fs_handle,
 
 	rc = dfs_release(ie->ie_obj);
 	if (rc) {
-		DFUSE_TRA_ERROR(dfs, "dfs_release() failed: (%s)", strerror(rc));
+		DFUSE_TRA_ERROR(dfs, "dfs_release() returned: %d (%s)", rc, strerror(rc));
```

Comment:
here it's the opposite. failed is more appropriate than returned.

Reply:
I've reverted this working change as well.